docs/projects/ad7616_sdz/index.rst

```diff
@@ -120,21 +120,21 @@ LK41               A         Onboard 3v3 power supply selected
 IP list
```

Comment:
This `IP list` section should've been removed in the previous PR, but I missed it somehow...